src/playground/sample_data_tooling/sample_data_tests/sample_data_plugin_tests/ad_plugin_data_config_tests/unit-test/data_trend_interface_unit_test.py

```diff
@@ -0,0 +1,32 @@
+"""
+Copyright OpenSearch Contributors
+SPDX-License-Identifier: Apache-2.0
+"""
+
+from datetime import datetime
+import pytest
+import sys
+import os
+
+# Adds the directory "/sample_data_tooling" to sys.path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))))
```

Comment:
Same as previous comments, can we make this simple? Thanks

Reply:
Done, made `sys.path` more readable.